packages/screens/AdministrationDashboard/component/GuardiansList.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { Image, TouchableOpacity, ViewStyle } from "react-native";
+import Popover from "react-native-popover-view";
+
+import { TooltipContant } from "./TooltipContant";
+import avaPNG from "../../../../assets/default-images/ava.png";
+import checkBadgeSVG from "../../../../assets/icons/check-badge.svg";
+import { BrandText } from "../../../components/BrandText";
+import { SVG } from "../../../components/SVG";
+import { neutral44, neutral11 } from "../../../utils/style/colors";
+import { fontSemibold16 } from "../../../utils/style/fonts";
+import { layout } from "../../../utils/style/layout";
+
+export const GuardiansList = ({ index }: { index: number }) => {
```

Comment:
Please type with `FC` (from "react")
```suggestion
export const GuardiansList: FC<{ index: number }> = ({ index }) => {
```